docs/docs/configuration/settings.md

```diff
@@ -31,6 +31,8 @@ The configuration file, `configuration.json` is in the Oni2 directory, whose loc
 
 - `editor.fontFamily` __(_string_)__ - The font family used by the editor surface. This must be a monospace font. The font may be specified by either the name of the font, or an absolute path to the font file.
```

Comment:
There's a good chance that absolute path feature is a potential bug pool. See my comment on the test case below for further explanations